core/sail/nativerdf/src/main/java/org/eclipse/rdf4j/sail/nativerdf/TxnStatusFile.java

```diff
@@ -13,41 +13,70 @@
 
 import org.eclipse.rdf4j.common.io.NioFile;
 
+import static java.nio.charset.StandardCharsets.US_ASCII;
+
 /**
  * Writes transaction statuses to a file.
  */
 class TxnStatusFile {
+	public static final byte NONE_BYTE = (byte) 0b00000001;
+	public static final byte ACTIVE_BYTE = (byte) 0b00000010;
+	public static final byte COMMITTING_BYTE = (byte) 0b00000100;
+	public static final byte ROLLING_BACK_BYTE = (byte) 0b00001000;
+	public static final byte UNKNOWN_BYTE = (byte) 0b00010000;
```

Comment:
Can these be moved into the enum? And can they be made private?

Reply:
Done.